shared-kernel-domain/src/main/java/org/dcsa/skernel/domain/persistence/entity/base/BaseTransportCall.java

```diff
@@ -26,6 +21,7 @@
 @MappedSuperclass
 @Table(name = "transport_call")
 @EqualsAndHashCode
+@Builder
```

Comment:
I'm pretty sure SuperBuilder only works with SuperBuilder and not with Builder

Reply:
indeed, fixed